### ISSUE TITLE
can't find url

### ISSUE THREAD

**Issue by xlf655** [bethrobson/Head-First-HTML#3]
in chapter14 browser can't find http://starbuzzcoffee.com/processorder.php

**Comment by bethrobson** [comment]
It's there!

**Comment by Victor-Kas** [comment]
No the link keeps redirecting to the book's  O'reilly  page

**Comment by bethrobson** [comment]
Yes, sorry they just removed that page with the PHP script;  we are in the process of moving the domain over to us so we can get it back up. Hopefully it won't be more than a few days!! Will post again once it's up. Sorry about that.

**Comment by Victor-Kas** [comment]
That was quick...... OK thank you.

**Comment by bethrobson** [comment]
Back up!! :-)  Still propagating through DNS but should be there in a few hours for everyone.
